app/src/main/res/values/strings.xml

```diff
@@ -653,4 +654,11 @@ be temporary, and you can try again later.</li>
     This is displayed during the authentication flow as the app launches.-->
     <string name="biometric_auth_moved_too_quickly">Finger moved too fast. Try again.</string>
 
+    <string name="enable_search_suggestion_title">Would you like to turn on search suggestions?</string>
+    <string name="enable_search_suggestion_subtitle">To get search suggestions, %1$s needs to send what you type in the address bar to the search engine you\'ve chosen. %2$s.</string>
```

Comment:
A comment in front of a string show up in Pontoon and can help a translator understand in what context a string is shown and what it means. Especially for strings with placeholder it's helpful to explain what the placeholder will be replaced with and maybe give an example value.

Also below for "Yes" and "No" it may be helpful for translators to know more about the context because this may influence how "yes" is going to be translated.